required/latex-lab/latex-lab-block.dtx

```diff
@@ -390,7 +392,7 @@
 %              {Instance of type \texttt{item} to be used to format
 %               the label string}{basic}
 %  \TemplateKey{item-skip}{skip}{The space in front of an item in the
-%               list.\fmi{May need to be on a different template level}}{\cs{itemsep}}
+%               list. If not specified the value specified in the block template instance is used}{}
```

Comment:
Unpaired braces due to removal of `\fmi{`.

Reply:
sorry don't see what you mean